### ISSUE TITLE
broken link in sidebar

### ISSUE THREAD

**Issue by deven-gqc** [Nixtla/neuralforecast#388]
can't click on `Contributing`
![image](https://user-images.githubusercontent.com/86305049/210422043-55d8d4ae-f205-41ed-86ab-7ac26163e854.png)

**Comment by kdgutier** [comment]
Hi @deven-gqc,

Thanks for reporting the broken link.
Here is the link to the contributing recommendations:
https://github.com/Nixtla/neuralforecast/blob/main/CONTRIBUTING.md

**Comment by deven-gqc** [comment]
@kdgutier yeah, I was able to find the file, I wanted to create a PR for it, but it seems like a weird quarto thing that I can't figure out

**Comment by kdgutier** [comment]
Sorry for the inconvenience @deven-gqc 

Here is a bit more detailed contribution instructions from the StatsForecast package:
https://github.com/Nixtla/statsforecast/blob/main/CONTRIBUTING.md

I think executing in the console `nbdev_preview` installs the quarto dependencies

![Screenshot 2023-01-03 at 1 58 18 PM](https://user-images.githubusercontent.com/19935241/210423326-2d247fe5-a263-42d9-911f-2a9e5fd876e9.png)
